### PR TITLE
Integrate install_upgrade job with origin and openshift-ansible merge queue

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov PTAL

### CODE REVIEW

File: config/common/test_suites/origin_fast.yml [hunk 1]
```diff
@@ -3,5 +3,6 @@ children:
   - "test_pull_request_origin_check"
   - "test_pull_request_origin_integration"
   - "test_pull_request_origin_extended_conformance_install"
+  - "test_pull_request_origin_extended_conformance_install_upgrade"
   - "test_pull_request_origin_extended_conformance_gce"
-  - "test_pull_request_origin_extended_networking_minimal"
\ No newline at end of file
+  - "test_pull_request_origin_extended_networking_minimal
```
Comment (stevekuznetsov) [inline config/common/test_suites/origin_fast.yml]: Missing an ending `"` here

File: config/common/test_suites/origin_fast.yml [hunk 2]
```diff
@@ -3,5 +3,6 @@ children:
   - "test_pull_request_origin_check"
   - "test_pull_request_origin_integration"
   - "test_pull_request_origin_extended_conformance_install"
+  - "test_pull_request_origin_extended_conformance_install_upgrade"
```
Comment (stevekuznetsov) [inline config/common/test_suites/origin_fast.yml]: @smarterclayton just want to confirm that we _are_ committing to running six jobs per PR test now

File: config/test_suites/merge_pull_request_openshift_ansible.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+children:
+  - "test_branch_origin_extended_conformance_install_with_status_check"
```
Comment (stevekuznetsov) [inline config/test_suites/merge_pull_request_openshift_ansible.yml]: These should come from `test_pull_request_openshift_ansible...` not Origin branch

File: config/common/test_suites/origin_fast.yml (hunk 1 above)
Comment (jhadvig) [inline config/common/test_suites/origin_fast.yml]: .. forgot to push

**Comment by smarterclayton** [comment]
It's ok for now, we probably need to make some of these optional
(non-blocking) which can wait for prow.

**Comment by jhadvig** [comment]
:+1:

**Pull request merged**